src/git-istage/ConsoleCommand.cs

```diff
@@ -24,5 +35,16 @@ public bool MatchesKey(ConsoleKeyInfo keyInfo)
         {
             return _key == keyInfo.Key && _modifiers == keyInfo.Modifiers;
         }
+
+        public string GetCommandShortcut()
+        {
+            string key = _key.ToString().Replace("Arrow", "");
+            if (_modifiers != 0)
+            {
+                return $"{_modifiers.ToString().Replace("Control", "CTRL")} + {key.ToString()}";
```

Comment:
I'd use mixed cases for `Ctrl` as that's consistent with `Alt` and `Shift`. Also, you may want to handle the output of digits (which renders `D1` instead of `1`).

```suggestion
                return $"{_modifiers.ToString().Replace("Control", "Ctrl")} + {key.ToString()}";
```